packages/core/src/new-api/internal/new-execution/types/jsonrpc.ts

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
All of this types used to store things in raw json-rpc format (e.g. numbers were 0x encoded strings). Now I changed them to more idiomatic types (e.g. numbers are `bigint`).